src/docs/development/add-to-app/ios/project-setup.md

```diff
@@ -7,6 +7,8 @@ description: Learn how to integrate a Flutter module into your existing iOS proj
 Flutter can be incrementally added into your existing iOS
 application as embedded frameworks.
 
+For examples, see the [add_to_app code samples][].
```

Comment:
see the iOS directories in the [add_to_app code samples][]?

Reply:
👍